src/overview/structure.md

```diff
@@ -1,13 +1,21 @@
 # Structure
 
-Every application you deploy on Platform.sh is built as a *virtual cluster*, containing a series of containers.  The master branch of your Git repository is always deployed as a production cluster.  Any other branch can be deployed as a development cluster.  By default you can have up to three live development clusters at once but you can buy more on a per-project basis.
+Every application you deploy on Platform.sh is built as a *virtual cluster*, containing a set of containers.  The master branch of your Git repository is always deployed as the production cluster.  Any other branch can be deployed as a development cluster.  By default, you can have up to three live development clusters at once, but you can buy more on a per-project basis.
 
-There are three types of containers within your cluster: One *Router*, one or more *Applications*, and zero or more *Services*.  When deployed, they conceptually look like this:
+There are three types of containers within your cluster:
+- one *Router*
+- one or more *Application* containers
+- zero or more *Service* containers
```

Comment:
By convention bulleted lists always use *, not -.

Also, put a newline between the preceding line and the list.

Reply:
Fixed.